src/validation.cpp

```diff
@@ -1161,7 +1166,7 @@ bool ReadBlockFromDisk(CBlock& block, const FlatFilePos& pos, const Consensus::P
     }
 
     // Check the header
-    if (!CheckProofOfWork(block.GetPoWHash(), block.nBits, consensusParams))
+    if (fCheckPOW && !CheckProofOfWork(block.GetPoWHash(), block.nBits, consensusParams))
         return error("ReadBlockFromDisk: Errors in block header at %s", pos.ToString());
```

Comment:
This says "ReadBlockOrHeader:" currently in 1.14, which seems better than referring to the old function name.

Reply:
Done